src/ddmd/mtype.d

```diff
@@ -9171,3 +9171,19 @@ extern (C++) final class Parameter : RootObject
 
     extern (D) static immutable bool[SR.max + 1][SR.max + 1] covariant = covariantInit();
 }
+
+/***********************************************************
+ * For printing two types with qualification when necessary.
+ * Disambiguates the case where toChars() on each type match.
+ */
+const(char*)[2] toAutoQualChars(Type t1, Type t2)
```

Comment:
Still needs "Returns" and "Params".  See https://dlang.org/spec/ddoc.html

Reply:
Added `Returns`. I don't know anything meaningful to put for `Params`.